plugins/devlink/meson.build

```diff
@@ -0,0 +1,63 @@
+if host_machine.system() == 'linux'
+  cargs = ['-DG_LOG_DOMAIN="FuPluginDevlink"']
+  plugins += {meson.current_source_dir().split('/')[-1]: true}
+
+  # Find libmnl dependency
+  libmnl = dependency('libmnl', required: false)
```

Comment:
nit: I would rather this be declared in the top level meson.build (that's what we usually do even if one plugin is the only one that needs it).

Reply:
Okay, added to dependencies.